packages/web-app-files/src/composables/actions/files/useFileActionsDownloadArchive.ts

```diff
@@ -59,6 +59,28 @@ export const useFileActionsDownloadArchive = ({ store }: { store?: Store<any> }
       })
   }
 
+  const archiverLimitsExceeded = (resources: Resource[]) => {
+    const archiverCapabilities = archiverService.capability
+    if (!archiverCapabilities) {
+      return
+    }
+
+    const selectedFilesAmount = resources.length
+
```

Comment:
Couldn't we early return here, so we don't need to fire the accumulator if the limit is already reached ?

Reply:
Surely, good idea :)